AMPDemos/index.html

```diff
@@ -310,100 +327,125 @@ <h2>Output</h2>
   </div>
 
   <script>
-  var RUNS = 1;
-  var AMP_BATCH_URL = 'https://acceleratedmobilepageurl.googleapis.com/v1/ampUrls:batchGet';
-  var AMP_BATCH_API_KEY = 'AIzaSyDteMhzKv7HPdYx2zV4nyiQ3bbCdDoTMbM';
-
-  var dbPromise = idb.open('AMPDemos', 3, (function(upgradeDb) {
-    if (!upgradeDb.objectStoreNames.contains('Tests')) {
-      var queuedRequests = upgradeDb.createObjectStore('Tests', {
-        unique: true,
-        autoIncrement: true,
-        keyPath: 'id',
-      });
+
+    let dropdown = document.getElementById('location-dropdown');
+    dropdown.length = 0;
+    let defaultOption = document.createElement('option');
+    defaultOption.text = 'Choose Server Location';
+    dropdown.add(defaultOption);
+    dropdown.selectedIndex = 0;
+    const url = 'https://www.webpagetest.org/getLocations.php?f=json&k=A';
+    const request = new XMLHttpRequest();
+    request.open('GET', url, true);
+    request.onload = function () {
+      if (request.status === 200) {
+
+        const data = JSON.parse(request.responseText).data;
+        const keys = Object.keys(data);
+        console.log(Object.keys(data).length);
+
+        let option;
+
+        for (let i = 0; i < keys.length; i++) {
+          option = document.createElement('option');
+          option.text = data[keys[i]].Label;
+          option.value = data[keys[i]].location;
+          dropdown.add(option);
+        }
+      } else {
+        // Reached the server, but it returned an error. Dulles & Sydney will be shown by default
+        console.log("Error fetching WPT server locations");
+
+      }
     }
-    if (!upgradeDb.objectStoreNames.contains('API_Keys')) {
```

Comment:
what's the reasoning for this delete?

Reply:
@noodles

Reply:
It wasn't actually deleted, just shifted down because of the formatting changes.